admin-dev/themes/new-theme/js/pages/import-data/ImportDataPage.js

```diff
@@ -48,6 +48,6 @@ export default class ImportDataPage {
 
     let importer = new Importer();
 
-    importer.import();
+    importer.import($('.import-form').serializeArray());
```

Comment:
watch out with this selector, can be dangerous, it's really simple and we can have many `.import-form`

Reply:
but this js file should be included only where it's needed - in this case only in import page, do you think it can still be dangerous in this case?